adminer/processlist.inc.php

```diff
@@ -51,6 +51,25 @@
 	echo ($i + 1) . "/" . lang('%d in total', max_connections());
 	echo "<p><input type='submit' value='" . lang('Kill') . "'>\n";
 }
+if (isset($_GET["showfull"])) {
+	echo "<p><a href='" . h(ME . "processlist=") . "'>" . lang('Hide full') . "</a>";
+} else {
+	echo "<p><a href='" . h(ME . "processlist=&showfull=") . "'>" . lang('Show full') . "</a>";
+}
 ?>
 <input type="hidden" name="token" value="<?php echo $token; ?>">
 </form>
+<?php if (isset($_GET["autorefresh"]) && !$error) { ?>
+<a href="<?php echo remove_from_uri("autorefresh"); ?>"><?php echo lang('Stop auto refresh'); ?></a>
```

Comment:
This is still not in `xx.inc.php`. You can add all new messages there by running `lang.php xx`.